doc/Usage.adoc

```diff
@@ -68,3 +68,12 @@ that if you have NEO Manager installed on multiple computers, the name
 change will only be reflected on the computer the change was made on.
 
 image:image4.PNG[image]
+
+
+=== U2F+CCID not working on Linux
+
+For Linux distributions with older libccid version, U2F+CCID mode will not

```

Comment:
It's not just CCID+U2F. Depending on what version of libccid you have it will support all, some, or none of the available modes.